cmd/apiserver/app/server/options.go

```diff
@@ -23,23 +23,20 @@ import (
 	"github.com/spf13/pflag"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	genericserveroptions "k8s.io/apiserver/pkg/server/options"
-
-	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/server"
 )
 
 const (
 	// Store generated SSL certificates in a place that won't collide with the
 	// k8s core API server.
 	certDirectory = "/var/run/kubernetes-service-catalog"
 
-	storageTypeFlagName = "storageType"
+	storageTypeFlagName = "storage-type"
```

Comment:
this doesn't change the flag, as far as I can tell. We just weren't using it before.

Reply:
It wasn't being used before, but it is now - see https://github.com/kubernetes-incubator/service-catalog/pull/2266/files#diff-5dd3f709c02872b9414e257945813685R89 and following lines.